changelog.md

```diff
@@ -277,6 +277,7 @@
 - Tuple expressions are now parsed consistently as
   `nnkTupleConstr` node. Will affect macros expecting nodes to be of `nnkPar`.
 
+- `nimscript` now accepts arbitrary file extensions.
```

Comment:
```suggestion
- `nim e` now accepts arbitrary file extensions for the nimscript file,
  although `.nims` is still the preferred extension in general.
```

first line because nimscript can appear in other contexts, but we don't accept the custom extension there.
2nd line because it's clearer when extension is present to document file type, plays better with syntax highlight (including on github), and extension-less files are sometimes gitignored, depending on what's in your gitignore of course